pkg/diffs/diffs.go

```diff
@@ -2,21 +2,27 @@ package diffs
 
 import (
 	"fmt"
+	"path/filepath"
 
 	"k8s.io/apimachinery/pkg/api/equality"
 	prowconfig "k8s.io/test-infra/prow/config"
 )
 
+const configInRepoPath = "cluster/ci/config/prow/config.yaml"
```

Comment:
This as well -- would love to see it come in with flags

Reply:
With these I am not that sure. These are just paths into `openshift/release` directory, to which the tool is already tied to anyway, so I do not see a problem with hardcoding these because they won't change often.

We already need three such paths and we will need more once we start handling templates, which would be four paths to `openshift/release` which we would pass as parameters but they are basically constants.